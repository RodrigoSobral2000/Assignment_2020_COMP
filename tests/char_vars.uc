char teste='\n';
char teste='\t';
char teste='\ooo';
char teste='\oooo';
char teste='\oo;
char uau= "ok;

char teste=''';