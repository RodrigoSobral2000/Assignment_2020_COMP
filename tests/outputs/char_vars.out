Line 1, col 2: invalid char constant (\\\')
CHRLIT('\\')
Line 3, col 2: invalid char constant (\')
Line 4, col 2: invalid char constant (\\\\')
Line 5, col 2: invalid char constant (\\\\\\\')
CHRLIT('\\')
Line 7, col 2: unterminated char constant
Line 8, col 2: unterminated char constant
Line 9, col 2: unterminated char constant
Line 11, col 2: unterminated char constant
CHRLIT('\\')
